src/i18n/en/translation.json

```diff
@@ -60,7 +60,9 @@
     "confirm": "Confirm",
     "ip_address": "IP address",
     "password_confirmation_required": "Password confirmation required",
-    "password_confirmation_required_description": "Please confirm your password to proceed"
+    "password_confirmation_required_description": "Please confirm your password to proceed",
+    "automated": "Automated",
+    "automated_tooltip": "This entry is managed by the system and cannot be modified."
```

Comment:
Not sure this string will be used so often to put it inside `common`. If we want to keep it here, I'd rephrase "This entry" to "This item", otherwise let's move it to port forward section and let's show "This port forward..."